specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -16,119 +17,113 @@ enum TextCategory {
 }
 
 #suppress "@azure-tools/typespec-azure-core/documentation-required" "MUST fix in next update"
-@doc("Image analyze category")
+@doc("Image analyze category.")
 enum ImageCategory {
   Hate,
   SelfHarm,
   Sexual,
   Violence,
 }
 
-@doc("The analysis request of the text.")
+@doc("The type of text analysis output.")
+enum AnalyzeTextOutputType {
+  @doc("Output severities in four levels, the value could be 0,2,4,6.")
```

Comment:
Since this is public documentation,

```suggestion
  @doc("Output severities in four levels, the value could be 0, 2, 4, or 6.")
```

Same below.

Reply:
Thanks heaths for the comments. This model is not introduced in change of this PR, but I will fix it .
cc : @mengaims  for awareness.

Reply:
Updated, please help review again.